app/views/settings/users/index.html.erb

```diff
@@ -10,6 +10,7 @@
     <%= content_tag :th, t(".local") if SiteSettings.federation_enabled? %>
     <%= content_tag :th, t(".fediverse_address") if SiteSettings.federation_enabled? %>
     <th><%= User.human_attribute_name(:roles) %></th>
+    <th><%= t(".quota.title") %></th>
```

Comment:
There doesn't seem to be a corresponding column the the table below for this heading?